dax/processors_v3.py

```diff
@@ -506,6 +506,8 @@ def build_singularity_cmd(self, runexec, command, var2val):
         # Append arguments for the singularity entrypoint
         cargs = command.get('args', None)
         if cargs:
+            # Unescape and then escape double quotes
```

Comment:
We've been jumping through 40 hoops to get the quoting right on these. How's this going to affect e.g. the below?

https://github.com/VUIIS/yaml_processors/blob/24ee862d76fdc413b4dea990ccf1fda23a9a6199/fmriprep_v21.0.1.yaml

Reply:
esp line 51

Reply:
This will only prepend the escapes if they are not already present. Which means that file could be less complex. Well, just one fewer escape.

Reply:
got it - thanks!